compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1786,4 +1786,23 @@ object messages {
             |If you want to create an ${"enum"} case, make sure the corresponding ${"enum class"} exists
             |and has the ${"enum"} keyword."""
   }
+
+  case class ExpectedTypeBoundOrEquals()(implicit ctx: Context)
+    extends Message(ExpectedTypeBoundOrEqualsID) {
+    val kind = "Syntax"
+    val msg =
+        hl"""`=', `>:', or `<:' expected"""
```

Comment:
Don't use backquotes. Interpolated strings will be highlighted. For example here:
```scala
val msg = hl"${"="}, ${">:"}, or ${"<:"} expected"
```